src/Microsoft.DotNet.ApiSymbolExtensions/Microsoft.DotNet.ApiSymbolExtensions.csproj

```diff
@@ -6,10 +6,7 @@
     <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">net8.0;net472</TargetFrameworks>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <!-- We pin the code analysis version when not in source build as we need to support running on older
-    SDKs when the OOB package is used. -->
-    <_MicrosoftCodeAnalysisVersion>4.0.1</_MicrosoftCodeAnalysisVersion>
-    <_MicrosoftCodeAnalysisVersion Condition="'$(DotNetBuildFromSource)' == 'true'">$(MicrosoftCodeAnalysisPackageVersion)</_MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisMinimumVersion>4.0.1</MicrosoftCodeAnalysisMinimumVersion>
```

Comment:
I noticed that referencing projects are getting CodeAnalysis-latest instead of the 4.0.1 version specified by ApiSymbolExtensions.  I think that's due to transitive pinning and not this PR, but I'll track it down.

Reply:
Yeah, confirmed.  Nightly builds are already regressed compared to RC1.  https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet8/NuGet/Microsoft.DotNet.ApiCompat.Task/overview/8.0.100-rc.2.23464.21